DataSetExplorer/Core/DataSets/Repository/DataSetDatabaseRepository.cs

```diff
@@ -23,25 +25,69 @@ public void Create(DataSet dataSet)
             _dbContext.SaveChanges();
         }
 
-        public DataSet GetDataSet(int id)
+        public DatasetDetailDTO GetDataSet(int id)
```

Comment:
If I understood correctly, it would mean that only the GetDataSetForExport method will be transferred to some "Write" repository. 
I removed a couple of unnecessary methods and moved some to the DataSetProjectDatabaseRepository, so I think this repository is now in better shape. Do you think there is still a need for segregation?